libvast/vast/query.hpp

```diff
@@ -71,6 +72,8 @@ struct query {
 
   enum class priority { normal, low };
 
+  uuid id = uuid::random();
```

Comment:
Can you add some comments to the data members of this class explaining what they are for?

Also, I think that the actor ID of the exporter actor may be better to use than generating a new random ID here. Otherwise `vast get 1 2 3` is gonna have three different query IDs, and I'm not sure that's the intended behavior.

Reply:
An actor can issue multiple queries, but it would not make sense for them to have the same IDs.